src/ansible/roles/packages/tasks/main.yml

```diff
@@ -5,6 +5,23 @@
   with_first_found:
   - files: '{{ ansible_distribution | distro_includes(ansible_distribution_major_version) }}'
 
+- name: Populate service facts
+  ansible.builtin.service_facts:
```

Comment:
Please, remove `ansible.builtin.` prefix, we don't have it on other places.

Reply:
Fixed in the latest version.